manifests/agent/iptables.pp

```diff
@@ -19,9 +19,8 @@
     }
   }
 
-  package { 'zabbix-agent_iptables':
+  package { $plugin_package:
     ensure  => present,
-    name    => $plugin_package,
     require => Package['zabbix-agent'],
```

Comment:
What do you gain with this change?

Reply:
Less lines.

Reply:
Reason for this syntax with additional param is so that you can address the package by the resource title, instead of relying on variable everywhere. I would like to keep it this way.

Reply:
Ok! Dismissing PR...